tests/fixtures/.env.eol

```diff
@@ -0,0 +1 @@
+VARIABLE_WITH_CRLF_EOL=123

```

Comment:
Please, make this test a code-only, no fixtures should be used. Since there are lots of editors / tools / etc that fix eofs.

Reply:
Just to make it clear - Are you suggest creating temp file inside test?

Reply:
yes